test/e2e/util_gatewayapi_test.go

```diff
@@ -580,6 +554,7 @@ func assertGatewaySuccessful(t *testing.T, namespace, name string) (*gatewayapiv
 		return false, nil
 	})
 	if err != nil {
+		t.Logf("Last observed gateway:\n%s", util.ToYaml(gw))
 		return nil, fmt.Errorf("gateway %s not %v, last recorded status message: %s", name, gatewayapiv1.GatewayClassConditionStatusAccepted, recordedConditionMsg)
```

Comment:
As mentioned in https://github.com/openshift/cluster-ingress-operator/pull/1152/files#r1994477437 , we can change `GatewayClassConditionStatusAccepted` to `GatewayConditionStatusAccepted`, to be more correct.

Reply:
Fixed in https://github.com/openshift/cluster-ingress-operator/compare/30b576c4293b2962b137bcb93ff77f278d7bf681..26a3918c7374cc16eac04b87c9e4267591c304a5..